howto/application/create-application.md

```diff
@@ -48,7 +48,7 @@ resources:
 
 ## Using the dashboard
 
-To create an application using the dashboard, use the *Add application* button on the *Applications* view, enter the required and any optional details that you want to provide and select *Add application*. This screen also provides the option to customise your application manifest.
+To create an application using the dashboard, click *Create application* from the applications view, enter the required and any optional details that you want to provide and confirm with *Create application* again. This screen also provides the option to customise your application manifest.
```

Comment:
I would suggest calling all pages what they are instead of a "view". From a design & architecture perspective everything that exists in the main navigation is a page: Applications, Instances, Nodes, Registry and Configuration. In the dashboard, whenever we refer to a page we explicitly call it a page. So having unity in the documentation about that would be less confusing to the user imo 👐

Reply:
Sounds great to me, thank you Keirthana 🎊